### PR TITLE
Add in-line tabs and copy-paste functionality to docs

### LINKED ISSUE #1888: Use sphinx extensions for in-line tabs and copy & paste functionality
- We did this for Woodwork and should do it for Featuretools
  - https://github.com/alteryx/woodwork/commit/65f487c11fafc85705b6bd2c6c98a53be3c1637a
  - https://github.com/alteryx/woodwork/commit/1649dba308efe5aa248c39d632c50c9fdac6bab5
## Ask
- Add 3 sphinx extensions (markdown reader, in-line tabs, copy button)
- Update install page to use in-line tabs
- Add Spark and Scala install instructions to install page
- Verify code blocks now have a copy button

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #1888

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -43,19 +43,28 @@ Before starting major work, you should touch base with the maintainers of Featur
 * You will need to install Spark, Scala, and GraphViz to run all unit tests:
 
      **macOS** (use [Homebrew](https://brew.sh/)):
-     ```bash
+     ```console
      brew tap AdoptOpenJDK/openjdk
      brew install --cask adoptopenjdk11
-     brew install scala
-     brew install apache-spark
```
Comment (dvreed77) [inline contributing.md:50]: why are these removed? I think they are need for tests to complete

Comment (gsheni) [reply contributing.md:50]: We changed it to one command
`brew install scala apache-spark`

**Comment by gsheni** [comment]
@jeff-hernandez We need to fix the Add-ons section so the PyPI and Conda tab are right next to each other
Screen Shot 2022-02-14 at 5 09 21 PM

- Here is the correct example in Woodwork:
  - https://woodwork.alteryx.com/en/latest/install.html#add-ons

- Same problem with the Graphviz section:
  - https://feature-labs-inc-featuretools--1905.com.readthedocs.build/en/1905/install.html#installing-graphviz

File: contributing.md (hunk 1 above)
Comment (gsheni) [reply contributing.md:50]: Oh right. @jeff-hernandez We should change the above command to `brew install scala apache-spark graphviz`

Comment (jeff-hernandez) [reply contributing.md:50]: updated

**Comment by jeff-hernandez** [comment]
@gsheni [that section](https://github.com/alteryx/featuretools/pull/1905#issuecomment-1039623132) seems to be a result of using sphinx inline tabs version `2021.3.28b7` for python 3.7.

```
sphinx-inline-tabs==2021.3.28b7; python_version == '3.7'
```

Using python 3.8 for RTD fixes this issue.

File: docs/source/install.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Install
+
+Featuretools is available for Python 3.7, 3.8, and 3.9. It can be installed from pip, [conda-forge](https://anaconda.org/conda-forge/featuretools), or from source.
```
Comment (gsheni) [inline docs/source/install.md:3]:
```suggestion
Featuretools is available for Python 3.7, 3.8, and 3.9. It can be installed from [pypi](https://pypi.org/project/featuretools/), [conda-forge](https://anaconda.org/conda-forge/featuretools), or from source.
```

File: docs/source/install.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+To install Featuretools, run the following command:
+
+````{tab} PyPI
+```console
+$ python -m pip install featuretools
```
[the remark this answers is not in the document]
Comment (jeff-hernandez) [reply docs/source/install.md:9]: updated

**Review by gsheni — APPROVED** [review]
